app/lib/frontend/static_files.dart

```diff
@@ -172,30 +158,32 @@ class StaticUrls {
     };
   }
 
-  /// A hashed version of the static assets referenced in [hashedFiles].
+  /// A hashed version of the static assets.
```

Comment:
Is this a mapping from something to something? Could we document what it maps from and to?